src/components/block/ProgramCard/ProgramCard.js

```diff
@@ -1,15 +1,13 @@
-import classNames from "classnames/bind";
+import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
 
 import s from "./ProgramCard.module.css";
 
 const ProgramCard = ({ theme, image, imageAlt, title, description }) => {
-  const classStyles = classNames.bind(s);
-  const backgroundTheme = classStyles({
-    programCardContainer: true,
-    light: theme === "light",
-    dark: theme === "dark",
+  const backgroundTheme = classNames(s.programCardContainer, {
+    [s.light]: theme === "light",
+    [s.dark]: theme === "dark",
```

Comment:
I don't think there should be two strict equalities for light and dark since only one can exist.  However, classnames does not allow terneries. There is also another component in this file that has a light and dark activation at src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.js on line 39 which is  [s.blackBackground]: blackBackground . Perhaps this can go in right now but there should be a standard for activation of light and dark.

Reply:
Actually, I think you can address @davidagustin's comment by just using the `theme` prop directly to access the associated CSS class as a variable property on the `s` object. Does it work if you write it as:

```javascript
classNames(s.programCardContainer, s[theme])
```
?